scripts/get-env-venue-owners.ts

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env node -r esm -r ts-node/register
+
+import fs from "fs";
+import { resolve } from "path";
+
+import admin from "firebase-admin";
+
+import { initFirebaseAdminApp, makeScriptUsage } from "./lib/helpers";
+
+const usage = makeScriptUsage({
+  description: "Print venue owners' email addresses.",
+  usageParams: "PROJECT_ID [CREDENTIAL_PATH]",
+  exampleParams: "co-reality-map [theMatchingAccountServiceKey.json]",
```

Comment:
there is a way to call the script without specifying the project id, just giving the service key. Please, check how can you do that in yours too (you can search for that in other scripts like scripts/add-auditorium-sections.ts or create-users.ts). 🙃 that would be more comfortable.

Reply:
fixed bb435db